modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJspGetPortletGroupIdMethodCheck.testjsp

```diff
@@ -0,0 +1,40 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/panel/init.jsp" %>
+
+<%
+List<PanelCategory> childPanelCategories = (List<PanelCategory>)request.getAttribute("liferay-application-list:panel:childPanelCategories");
+PanelCategory panelCategory = (PanelCategory)request.getAttribute("liferay-application-list:panel:panelCategory");
+%>
+
+<c:if test="<%= !childPanelCategories.isEmpty() %>">
+
+	<%
+	PanelAppRegistry panelAppRegistry = (PanelAppRegistry)request.getAttribute(ApplicationListWebKeys.PANEL_APP_REGISTRY);
+
+	for (PanelApp panelApp : panelAppRegistry.getPanelApps(panelCategory.getKey())) {
+	%>
+
+		<c:if test="<%= panelApp.isShow(permissionChecker, themeDisplay.getScopeGroupId()) %>">
+			test
+		</c:if>
+
+	<%
+	}
+	%>
+
+</c:if>
```

Comment:
@TamyrisTorres, please keep the test scope as simple as possible, like the test file related with the Java file (modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaGetPortletGroupIdMethodCheck.testjava)